addons/mail/static/src/models/chat_window_header_view.js

```diff
@@ -2,10 +2,32 @@
 
 import { registerModel } from '@mail/model/model_core';
 import { one } from '@mail/model/model_field';
+import { isEventHandled } from '@mail/utils/utils';
 
 registerModel({
     name: 'ChatWindowHeaderView',
     identifyingFields: ['chatWindowOwner'],
+    recordMethods: {
+        /**
+         * @param {MouseEvent} ev
+         */
+        onClick(ev) {
+            if (
+                isEventHandled(ev, 'ChatWindow.onClickCommand') ||
+                isEventHandled(ev, 'ChatWindowHeader.ClickShiftNext') ||
+                isEventHandled(ev, 'ChatWindowHeader.ClickShiftPrev') ||
+                isEventHandled(ev, 'ChatWindow.onClickHideMemberList') ||
+                isEventHandled(ev, 'ChatWindow.onClickShowMemberList')
+            ) {
+                return;
+            }
+            if (!this.chatWindowOwner.isVisible) {
+                this.chatWindowOwner.onClickFromChatWindowHiddenMenu();
+            } else {
+                this.chatWindowOwner.onClickHeader();
```

Comment:
Having a handler that calls another handler feels a bit odd to me. I'd recommend either moving the code of `onClickFromChatWindowHiddenMenu` and `onClickHeader` in here, or renaming it to something more declarative that would make the extra indirection worthwhile.

Reply:
How about the second suggestion?

Reply:
Can you provide example? For me, these methods are specificity code for the handler, so keeping the `onX` naming convention still make sense.